py/desispec/workflow/desi_proc_funcs.py

```diff
@@ -560,7 +570,7 @@ def create_desi_proc_batch_script(night, exp, cameras, jobdesc, queue, nightlybi
         else:
             if jobdesc.lower() in ['science','prestdstar']:
                 fx.write('\n# Do steps through skysub at full MPI parallelism\n')
-                srun = f'srun -N {nodes} -n {ncores} -c {threads_per_core} {cmd} --nofluxcalib'
+                srun = f'srun -N {nodes} -n {ncores} -c {threads_per_core} --cpu-bind=cores {cmd} --nofluxcalib'
```

Comment:
Line 567 above is another `srun` where you haven't added `--cpu-bind`. It pertains to arcs and flats. Was that intentional?

Reply:
Full disclosure: I wasn't sure what the right answer was in that case and not specifying it was working so I left it alone.  We can revisit that one later.